app/views/reports/dashboards/_report_filter.haml

```diff
@@ -15,6 +15,8 @@
         = f.input :match_route, as: :select_2, collection: @filter.match_route_options_for_select,  selected: @filter.match_routes&.first, include_blank: false
     .col-4
       = f.input :program_types, as: :select_2, collection: @filter.program_type_options_for_select, input_html: { multiple: true, placeholder: 'Any Program Type', data: { placeholder: 'Any Program Type' } }
+    .col-4
```

Comment:
Can you update these all to `.col-3` so we end up with 4 items on the row to match the client-level filters below?
<img width="1213" alt="Screenshot 2024-08-27 at 10 26 22 AM" src="https://github.com/user-attachments/assets/2e38d73a-c3c6-4791-a233-587550902777">

Reply:
Will include this in the neext commit